pkg/pillar/cmd/nodeagent/nodeagent.go

```diff
@@ -281,7 +281,7 @@ func Run(ps *pubsub.PubSub) {
 		AgentName:     "baseosmgr",
 		TopicImpl:     types.ZbootStatus{},
 		Activate:      false,
-		Ctx:           &nodeagentCtx,
+		Ctx:           &nodeagentCtxPtr,
```

Comment:
Do you really want a pointer to a pointer? I suspect this will get a panic due to a type assertion on boot (make run).